playbooks/loadbalancer_deploy.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
@Smithe2413 can you rename this playbook to be something like `configure_lb_capsuels.yml`